src/bundle/Resources/public/js/scripts/admin.location.change.language.js

```diff
@@ -1,10 +1,10 @@
-(function (global, doc) {
+(function(global, doc) {
     const changeLocationLanguage = (event) => {
         window.location = event.currentTarget.value;
     };
-    const locationLanguageSwitcher = doc.querySelector('.ez-location-language-change');
+    const locationLanguageSwitchers = [...doc.querySelectorAll('.ez-location-language-change')];
```

Comment:
We don't need this notation anymore: `[...doc.querySelectorAll('.ez-location-language-change')];`.
Now, the notation `doc.querySelectorAll('.ez-location-language-change');` is just enough when the issue with Edge browser has been fixed by the browser developers :)